api/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM elixir:1.14.4
```

Comment:
suggestion: we generally use the containers from `hexpm/elixir`, which allow us to specify the Elixir, Erlang, and Linux versions we're using.